client/src/pages/FlightData/tabs/FlightPlan/FlightPlanToolbar.js

```diff
@@ -277,83 +222,69 @@ const FlightPlanToolbar = props => {
 				</Row>
 				<span>Place Point:</span>
 				<Row columns="minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr) minmax(0, 2fr)  minmax(0,1fr)">
-					<Box
-						ref={input1Ref}
-						type="text"
-						editable={greyOutPlacePoint()}
-						placeholder={greyOutPlacePoint() ? "Lat":"Disabled"}
-						content={greyOutPlacePoint() ? lat:""}
-						onChange={(e)=>setLat(e)}
-						style={{"backgroundColor":"#E2DBD5","borderColor":"transparent" }}
-						onKeyDown={(e) => 
-							enterPlacePoint(e)
-						}
+					<PlacePointInputBox
+						refProp={input1Ref} 
+						editable={()=>greyOutPlacePoint()} 
+						placeholder={greyOutPlacePoint() ? "Lat":"Disabled"} 
+						content={()=>greyOutPlacePoint() ? lat:""} 
+						onChange={(e) => setLat(e)} 
+						style={{"backgroundColor":"#E2DBD5","borderColor":"transparent" }} 
+						onKeyDown={(e) => enterPlacePoint(e)}
 					/>
```

Comment:
you're missing the point of wrapping this within its own component. you don't have to pass all those props down. they're redundant, and can just be put inside the component itself.